### DISCUSSION TITLE
Inline Editor SleekGrid

### DISCUSSION THREAD

**Discussion by reach2rv** [serenity-is/Serenity#7134] [Help]
Hi @VictorTomaili - 

Apologies to tag you. I need some clarification on sleekgrid inline edit. unfortunately, I am no more having startsharp subscription and therefore asking here, your support would be appreciated.

How I can handle before edit event, and get which editor is used. for example: I have service lookup editor on row to which I want to cascade based on certain value in row. how this can be done?

Please help me with your valuable inputs.

Regards,

**Comment by VictorTomaili** [comment]
Sorry, inline editing is a pro feature. You need to ask in StartSharp repository for pro features. I can not provide support here. 

Best regards.
